package.json

```diff
@@ -21,25 +52,22 @@
     "name": "Serverless Workflow Specification Authors",
     "url": "https://serverlessworkflow.io"
   },
+  "contributors": [
```

Comment:
I am not sure about this since this is a community project. Any thought here @tsurdilo ?

Not sure if instead we should add you here https://github.com/serverlessworkflow/specification/blob/main/community/contributors.md 

Are you ok with that @JBBianchi ?

Reply:
Sure, no worries, tell me if it needs to be removed.

Reply:
you can do something like this: https://github.com/serverlessworkflow/vscode-extension/blob/main/package.json#L8

or add name: "Serverless Workflow Contributors" and url to serverlessworkflow.io

Reply:
I committed a fix